src/themes/theme.ts

```diff
@@ -56,6 +56,8 @@ export interface Theme {
     label?: {
       stroke?: ColorRepresentation;
       color: ColorRepresentation;
+      fontSize?: number;
+      yOffset?: number;
```

Comment:
Need to add docs to the props

Reply:
@amcdnl What do you mean? Comment with description above or something else?

Reply:
Yes